File: validator/javascript/src/printer.js
class Printer {

  static error(message) {
    const dataReturn = {'status': 'error', message };
    console.log(JSON.stringify(dataReturn));
    process.exit(0);
  }

  static success(result) {
    const dataReturn = {'status': 'success', result };
    console.log(JSON.stringify(dataReturn));
    process.exit(0);
  }
}

module.exports = {
  Printer
}

File: validator/python/src/printer.py
import sys
import json 

class Printer:

  @staticmethod
  def error(message):
    data_return = {
      "status": "error", 
      "message": message
    }
    
    print(json.dumps(data_return))
    sys.exit(0)


  @staticmethod
  def success(result):
    data_return = {
      "status": "success", 
      "result": result 
    }

    print(json.dumps(data_return))
    sys.exit(0)
